helm/values-prod.yaml

```diff
@@ -120,9 +120,9 @@ microservice-chart:
     MONGO_MIN_POOL_SIZE: "0"
     MONGO_MAX_POOL_SIZE: "50"
     MONGO_MAX_IDLE_TIMEOUT_MS: "600000"
-    MONGO_CONNECTION_TIMEOUT_MS: "2000"
+    MONGO_CONNECTION_TIMEOUT_MS: "5000"
```

Comment:
maybe we have to increase also `MONGO_WAITING_QUEUE_MS` accordingly to this value, what do you think?

Reply:
You're right, fixed!